goagen/gen_app/generator.go

```diff
@@ -372,6 +372,10 @@ func (g *Generator) generateMediaTypes() error {
 		codegen.SimpleImport("unicode/utf8"),
 		codegen.NewImport("uuid", "github.com/satori/go.uuid"),
 	}
+	for _, v := range g.API.MediaTypes {
+		var seen []*design.AttributeDefinition
+		imports = codegen.AttributeImports(v.AttributeDefinition, imports, seen)
```

Comment:
would have to restructure the entire AttributeImports function if we pass a nil seen here.